Mithril update 2024-03-20 (#363)
IntersectMBO/cardano-updates: blog/2024-03-20-mithril.md

---
title: Mithril Team Update
slug: 2024-03-20-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team kept implementing the certification of Cardano transactions in Mithril networks: they kept working on scaling the signature and proof generation for `mainnet`, completed the UI improvements of the explorer, provided metadata about the point of the chain used during the verification process, and investigated a bug in the blocks parser that prevents some Conway transactions to be signed. Additionally, they have started prototyping the implementation of a more versatile beaconing mechanism that will be used to sign transactions with low latency.

Reviewer: ```suggestion
This week, the Mithril team continued implementing the certification of Cardano transactions in Mithril networks. They focused on scaling the signature and proof generation for `mainnet`, completed UI improvements for the explorer, provided metadata about the point of the chain used during the verification process, and investigated a bug in the block parser that prevents some Conway transactions from being signed. Additionally, they began prototyping the implementation of a more versatile beaconing mechanism to sign transactions with low latency.
```

> Finally, the team started implementing some community requested feature to the output folder structure verifications made by the client, and fixed a source of flakiness in the end-to-end test in the CI.

Reviewer: ```suggestion
Finally, the team started implementing some community-requested features to verify the output folder structure made by the client, and fixed a source of flakiness in the CI end-to-end test.
```
